misc/mprocs.yaml

```diff
@@ -23,5 +23,7 @@ procs:
     shell: tail -n +0 -F $FM_LOGS_DIR/bitcoind.log
   devimint:
     shell: tail -n +0 -F $FM_LOGS_DIR/devimint.log
+  devimint-outer:
+    shell: tail -n +0 -F $FM_LOGS_DIR/devimint-outer.log
```

Comment:
I don't think we need both. Maybe just have `devimint` tab point at the output of calling `devimint` on the command line (which we call `devimint-outer.log` here). This file has strictly more information in it.